.dev/docker/env/Dockerfile

```diff
@@ -2,15 +2,15 @@
 # https://gitlab.com/nvidia/container-images/cuda/tree/master/dist
 # To enable cuda, use "--build-arg USE_CUDA=true" during image build process
 ARG USE_CUDA
-ARG CUDA_VERSION="9.0"
+ARG CUDA_VERSION="9.2"
 ARG UBUNTU_DISTRO="16.04"
-# Known conflicts:
-# * 9.1 is an issue with 16.04 for Eigen
 ARG BASE_CUDA_IMAGE=${USE_CUDA:+"nvidia/cuda:${CUDA_VERSION}-devel-ubuntu${UBUNTU_DISTRO}"}
 ARG BASE_IMAGE=${BASE_CUDA_IMAGE:-"ubuntu:${UBUNTU_DISTRO}"}
 
 FROM ${BASE_IMAGE}
 
+ARG USE_CUDA
```

Comment:
Why did you repeat L4 here?

Reply:
Not needed now, earlier I was checking if the flag was set to then install Eigen 3.3.7